src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -690,6 +781,11 @@ private static int GetIntQueryParameterValue(NameValueCollection queryStringName
         {
             IDurableEntityClient client = this.GetClient(request);
 
+            if (entityId.EntityKey.Equals("$"))

```

Comment:
Let's add a comment explaining why we're doing this - i.e. "$" means to return the default entity, which is the one with no entity key.